core/templates/dev/head/player/PlayerServices.js

```diff
@@ -302,6 +307,46 @@ oppia.factory('oppiaPlayerService', [
     }
   };
 
+  var _loadInitialInformationCardData = function() {
+
+    // TODO(sll): move these three handlers into a separate exp_metadata.py controller.
+    var infoCardDataUrl = '/createhandler/data/' +  _explorationId;
+    var infoCardSnapshotsUrl = '/createhandler/snapshots/' +  _explorationId;
+    var infoCardStatisticsUrl = '/createhandler/statistics/' +  _explorationId + '/all';
+
+    // This is needed to get exploration objective/goal, contributors
+    // and exploration tags.
+    $http.get(infoCardDataUrl).success(function(data) {
+      _explorationObjective = data.objective;
+      _explorationTags = data.tags;
+      // TODO(sll): in the backend, compute the actual contributors
+      // based on the list of commits to the exploration.
+      _explorationContributorUsernames = data.rights.owner_names.concat(data.rights.editor_names);
```

Comment:
In JS please try and keep line lengths to 80 chars where possible (failing that, 100 chars). This should be possible to set in the JSLint settings.